### DISCUSSION TITLE
How can I pass a raw sql string?

### DISCUSSION THREAD

**Discussion by stan023** [porsager/postgres#793] [Q&A] [answered]
I need to write a function and pass complete sql string to postgresjs, something like this:
```
async function query(sql_string){
    return await sql` ${sql_string} `
}
```
The above function doesn't work. How can I create such a function?

**Comment by porsager** [comment, marked as answer]
Check out .unsafe in the documentation ;) https://github.com/porsager/postgres#unsafe-raw-string-queries

**Reply by stan023** [reply]
Thank you it works! So why this function called "unsafe"? There is a "arguments" parameter for passing variables. It looks like a general query function.
